### PR TITLE
[DO NOT MERGE] VA List Api version edit

### PR DESCRIPTION
**Pull request by t-robalt**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by t-robalt** [comment]
@talhers @yaakoviyun

**Comment by dsgouda** [comment]
Reassigning due to some other deliverables

**Review by jaredmoo — APPROVED** [review]

File: specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/examples/DatabaseVulnerabilityAssessmentListByDatabase.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
         "subscriptionId": "00000000-1111-2222-3333-444444444444",
         "resourceGroupName": "vulnerabilityaseessmenttest-4799",
         "serverName": "vulnerabilityaseessmenttest-6440",
-        "databaseName": "testdb",
+		"databaseName": "testdb",
```
Comment (jianghaolu) [inline specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/examples/DatabaseVulnerabilityAssessmentListByDatabase.json]: indentation

**Comment by jianghaolu** [comment]
@t-robalt LGTM - but what's the purpose of deleting a bunch of examples?

**Comment by t-robalt** [comment]
@jianghaolu The List method of database and managed database moved to other api-versions, so their examples in 2018-060-01 are no longer necessary

**Review by jianghaolu — APPROVED** [review]

**Comment by dsgouda** [comment]
@jianghaolu @jaredmoo was this supposed to be merged?

**Comment by yaakoviyun** [comment]
Yep, the code should have reached production by now.

Sent from Outlook
________________________________
מאת: Deepak Shankargouda <notifications@github.com>
‏‏נשלח: יום חמישי 01 נובמבר 2018 19:42:54
‏‏אל: Azure/azure-rest-api-specs
עותק: Yaakov Iyun; Mention
‏‏נושא: Re: [Azure/azure-rest-api-specs] [DO NOT MERGE] VA List Api version edit (#4257)


@jianghaolu<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fjianghaolu&data=02%7C01%7Cyaiyun%40microsoft.com%7Cf9bae65aba284278574a08d640217436%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636766909755353960&sdata=sT%2F3DiwFoVj%2BjxS68QTHGc1tI9%2BAwy6OyrXecBztkKE%3D&reserved=0> @jaredmoo<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fjaredmoo&data=02%7C01%7Cyaiyun%40microsoft.com%7Cf9bae65aba284278574a08d640217436%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636766909755353960&sdata=XnAikyaA%2B0ClgkRm5qfvHFHErgBXAiwVPtuzgRF9KnQ%3D&reserved=0> was this supposed to be merged?

**Comment by dsgouda** [comment]
Thanks for confirming, I was concerned because of the title of the PR 👍

**Pull request merged**